illumio_connector.py

```diff
@@ -1,249 +1,341 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-# -----------------------------------------
-# Phantom sample App Connector python file
-# -----------------------------------------
+# File: illumio_connector.py
+#
+# Copyright (c) 2022 Illumio.
+#
+# This unpublished material is proprietary to Illumio.
+# All rights reserved. The methods and
+# techniques described herein are considered trade secrets
+# and/or confidential. Reproduction or distribution, in whole
+# or in part, is forbidden except by express written permission
+# of Illumio.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
 
-# Python 3 Compatibility imports
-from __future__ import print_function, unicode_literals
-
-import json
 
 # Phantom App imports
 import phantom.app as phantom
-import requests
-from bs4 import BeautifulSoup
-from phantom.action_result import ActionResult
 from phantom.base_connector import BaseConnector
 
-# Usage of the consts file is recommended
-from illumio_consts import *
-from illumio_core import *
+from phantom.action_result import ActionResult
 
+# Usage of the consts file is recommended
```

Comment:
Don't leave this comment in now that the context has been removed.

Reply:
Done